_posts/2020-07-24-when-scenarios-and-reality-diverge/when-scenarios-and-reality-diverge.Rmd

```diff
@@ -0,0 +1,380 @@
+---
+title: "When Scenarios and Reality Diverge"
```

Comment:
If you want, you may also add a short description. All other posts have a `description:` field.